proposed/2023/NuGetAudit-without-nuget.org-as-packageSource.md

```diff
@@ -0,0 +1,168 @@
+# NuGetAudit without nuget.org as a package source
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Author: [zivkan](https://github.com/zivkan)
+- GitHub Issue: [12698](https://github.com/NuGet/Home/issues/12698)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+This proposal adds a new `<auditSource>` to *NuGet.Config* files, allowing *NuGetAudit* to obtain known vulnerabilities without adding nuget.org as a package source.
+
+## Motivation
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+[NuGetAudit](https://learn.microsoft.com/nuget/concepts/auditing-packages) is a feature where NuGet will report during restore which packages with known vulnerabilities are being used by projects, [proposed in late 2022](../2022/vulnerabilities-in-restore.md), and finally released with .NET 8 in 2023.
+The first version of NuGetAudit used package sources as the data source for known vulnerabilities, which will be referred to as a Vulnerability Database (VDB) for the remainder of this document.
```

Comment:
FYI a better acronym for the database and related items might be:

OSV - Open source vulnerabilities -> https://osv.dev/

OSVDB - Open source vulnerability database -> https://github.com/advisories

Reply:
There's nothing in NuGet that requires the vulnerability information to be limited to open source libraries. If closed source package providers (I'm thinking the ASP.NET & WPF control vendors) want to provide advisories on their closed source packages, it'll work just fine.  I also don't want to want to give a false impression that NuGet's using one of these formats. We created our own, minimal, file format in order to maximize performance.